utils/efushell/integrationtest.json

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Using tabs for indent or four spaces (I can't tell)? Let's stick to spaces. I normally use 2 spaces just as we do for C++ code. Might be worth doing consistently for this repo?